VK_V1.0/VK_V1.0/UIView+extension.swift

```diff
@@ -0,0 +1,26 @@
+// UIView+extension.swift
```

Comment:
```suggestion
// UIView+Extension.swift
```
и сам файлик тоже переименовать